module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -463,10 +463,24 @@ public function getConfig($function, $params = [])
      */
     protected function isHoldable($locationName)
     {
-        return !in_array(
-            $locationName,
-            (array)($this->config['Holds']['excludeHoldLocations'] ?? [])
-        );
+        $mode = $this->config['Holds']['excludeHoldLocationsCompareMode'] ?? 'exact';
+        $excludeLocs = (array)$this->config['Holds']['excludeHoldLocations'] ?? [];
+
+        // Exclude checking by regex match
+        if ($mode == "regex") {
```

Comment:
Might be worth doing `trim(strtolower($mode))` just to be a little more tolerant of non-standard input (particularly since the comments in the file capitalize "Exact" and "Regex" in the headings above the example, which might lead someone to enter a capitalized version).